### ISSUE TITLE
Too Big loss

### ISSUE THREAD

**Issue by mipsan** [tensorflow/models#9892]
![image](https://user-images.githubusercontent.com/3205851/114335826-88eca600-9b88-11eb-94d9-98ae6d8e90e8.png)

As this image shows, it has too big loss value. 
And  saved_model.pb cannot predict properly.  Not a low accuracy, but failed with unfamiliar error messages.

TF2 : 2.4.1  
Python: 3.6.9
CUDA: 11.2 
CUDNN: 8.1

number of class: 1
pre-trained model: ssd-mobilenet_v2 

`label_map.pbtxt `has only one class as follow:
```
item {
    id: 1
    name: 'my_class'
}
```

Would you advise to my `pipiline.config` ?

Thanks,

[pipeline.config.txt](https://github.com/tensorflow/models/files/6293653/pipeline.config.txt)

**Comment by August0424** [comment]
I met the same problem. I've got no idea how to deal with it

**Comment by August0424** [comment]
Oh, I have got it. I forgot to normalie the bbox coordinates (xmin, xmax, ymin, ymax) by the width and height of input image when preprocessing data to tfrecord. This resulted in a big loss. After correcting my preprocessing code, it works normally.

**Comment by saikumarchalla** [comment]
@August0424  Closing the issue since the issue was resolved  for you. please  feel free to reopen the issue if you have any concern. Thanks!

**Comment by mipsan** [comment]
@August0424 , even this thread is closed, I would add sth more.

If the normalization is referred following, 
```
def create_tf_example(group, path):
    ...
    for index, row in group.object.iterrows():
        xmins.append(row['xmin'] / width)
        xmaxs.append(row['xmax'] / width)
        ymins.append(row['ymin'] / height)
        ymaxs.append(row['ymax'] / height)
        classes_text.append(row['class'].encode('utf8'))
        classes.append(class_text_to_int(row['class']))
    ...
```
it's included during the process. 

I tested couple of SSD-Mobilenet pre-trained models of Model zoo.
But I met the issue at the "_SSD MobileNet v2 320x320_", and did not at the both "_SSD MobileNet V2 FPNLite 320x320_" and "_SSD MobileNet V2 FPNLite 640x640_".

I can say that other parameters are almost same.
